2018/index.dd

```diff
@@ -21,6 +21,36 @@ $(LI $(B Under the hood) looks at compiler and standard library implementation a
 $(LI $(B Hot topics) such as safe deterministic memory management, migrating from other languages, or community engagement)
 $(LI $(B Research reports) by students doing research on or using the D language)
 )
+$(P 
+But don't be constrained! Originality in topics and formats is warmly encouraged. If it is relevant to the D language, it fits. Just mind the guidelines and don't forget the deadlines below.
+)
+$(P 
+We expect original work, or work synthesized from community folklore into interesting tidbits by the author. Related work can be mentioned with credit. Overviews of an area covered by other authors is acceptable, again with due credit and assuming the overview has intrinsic value.
+)
+$(T h2,,Selection Criteria)
+$(P 
+The committee will review and rate submissions according to the following criteria:
+)
+$(T ul,,
+$(LI $(B Relevance:) Is the topic interesting to DConf attendees? Will a coverage be beneficial? Is it building on tried and true ideas, or on the contrary, blazing new trails?)
+$(LI $(B Coverage:) Does the proposal cover the topic in depth? What are the main insights dispensed by the submission?)
+$(LI $(B Organization:) Is the submission well organized? Will the audience tune into the narrative?)
+$(LI $(B Novelty and Originality:) How new and noteworthy is the material? Previously published material is acceptable provided it hasn't reached a wide audience within the D community yet.)
+$(LI $(B Credentials:) What qualifies you to talk on the topic of choice? Although this criterion favors experienced and well-known submitters, we also very strongly encourage submissions from up-and-coming contributors who have accumulated "street cred" through their open source and forum contributions.)
+$(LI $(B Bottom line:) What's your session's takeaway? How is it going to improve attendees' knowledge, outlook, and inspiration?)
+)
+$(T h2,,Submission Guidelines)
+$(P We expect submission in plain text format or one of the usual rich document formats, such as PDF or DOC. Send your submissions to $(LINK2 mailto:dconf2015@digitalmars.com, dconf2015@digitalmars.com). The submission should include:)
```

Comment:
Is there `dconf18@dlang.org`? I can always setup redirects from `dlang.io` or `dlang.rocks` if wanted.